go/gossip/main/gossip_server.go

```diff
@@ -1,18 +1,56 @@
 package main
 
 import (
+    "errors"
 	"flag"
+    "fmt"
+    "io/ioutil"
 	"log"
 	"net/http"
+    "strings"
 
+	ct "github.com/google/certificate-transparency/go"
 	"github.com/google/certificate-transparency/go/gossip"
 )
 
 var dbPath = flag.String("database", "/tmp/gossip.sq3", "Path to database.")
 var listenAddress = flag.String("listen", ":8080", "Listen address:port for HTTP server.")
+var logKeys = flag.String("log_public_keys", "", "Comma separated list of files containing trusted Logs' public keys in PEM format")
```

Comment:
Am I right, or it might not skip junk between PEMs? Again, can be fixed up later, I was already thinking that the bit about the `clock` interface could have easily been in a separate PR... :wink: